### PR TITLE
refactor: drop TuiErrorModule

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by nsbarsukov — APPROVED** [review]

File: projects/core/components/error/error.component.ts [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+import {tuiIsString, TuiLetDirective, TuiValidationError} from '@taiga-ui/cdk';
 import {tuiFadeIn, tuiHeightCollapse} from '@taiga-ui/core/animations';
 import {TUI_ANIMATIONS_SPEED, TUI_DEFAULT_ERROR_MESSAGE} from '@taiga-ui/core/tokens';
 import {tuiToAnimationOptions} from '@taiga-ui/core/utils';
+import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
 
 @Component({
+    standalone: true,
     selector: 'tui-error',
+    imports: [NgIf, TuiLetDirective, AsyncPipe, PolymorpheusModule],
```
Comment (mdlufy) [inline projects/core/components/error/error.component.ts:18]: maybe to put `ngIf` and `AsyncPipe` firstly ?

File: projects/demo/src/modules/components/error/examples/1/index.ts [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
 import {TuiValidationError} from '@taiga-ui/cdk';
+import {TuiErrorComponent, TuiLabelDirective} from '@taiga-ui/core';
+import {TuiSwitchComponent} from '@taiga-ui/kit';
 
 @Component({
-    selector: 'tui-error-example-1',
+    standalone: true,
+    imports: [TuiLabelDirective, TuiSwitchComponent, TuiErrorComponent, FormsModule],
```
Comment (mdlufy) [inline projects/demo/src/modules/components/error/examples/1/index.ts:11]: maybe put `FormsModule` firstly ?

**Review by mdlufy — APPROVED** [review]

File: projects/core/components/error/error.component.ts (hunk 1 above)
Comment (splincode) [reply projects/core/components/error/error.component.ts]: fixed

File: projects/demo/src/modules/components/error/index.ts [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
-import type {TuiDocExample} from '@taiga-ui/addon-doc';
+import {TuiDemo} from '@demo/utils';
 import {TuiValidationError} from '@taiga-ui/cdk';
+import {TuiErrorComponent, TuiLinkDirective} from '@taiga-ui/core';
 
 @Component({
-    selector: 'example-tui-error',
-    templateUrl: './error.template.html',
+    standalone: true,
+    imports: [TuiDemo, TuiErrorComponent, TuiLinkDirective, RouterLink],
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/components/error/index.ts]: fixed

File: projects/demo/src/modules/components/error/examples/1/index.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/error/examples/1/index.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**